Ashwin/HTML/06_Assignment/06_Assignment.css

```diff
@@ -0,0 +1,21 @@
+.blue1 {
+    height: 50px;
+    width: 100%;
+    background-color: rgb(182, 190, 204);
+    text-align: right;
+    border-radius: 5px;
+}
+.blue2 {
```

Comment:
use .blue1 class for similar style and apply .blue2 class whenever needed with only height, font-size remove other styles from this class